temp.py

```
@@ -0,0 +1,8 @@
+import os
+
+def run_command(user_input):
+    # ⚠ Vulnérabilité : injection de commande système
+    os.system("echo " + user_input)
```

Ruff (0.12.2)

Starting a process with a shell, possible injection detected

(S605)